core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -143,13 +161,45 @@ public interface Configs {
           .longConf()
           .createWithDefault(2000L);
 
-  ConfigEntry<Long> KV_DELETE_AFTER_TIME =
+  ConfigEntry<Optional<Long>> KV_DELETE_AFTER_TIME =
       new ConfigBuilder(KV_DELETE_AFTER_TIME_KEY)
           .doc(
-              "The maximum time in milliseconds that the deleted data and old version data is kept")
-          .version(ConfigConstants.VERSION_0_3_0)
+              String.format(
+                  "The maximum time in milliseconds that the deleted data and old version data is kept, "
+                      + "max delete time allow is %s ms(30 days), "
+                      + "min delete time allow is %s ms(10 minutes)",
+                  MAX_DELETE_TIME_ALLOW, MIN_DELETE_TIME_ALLOW))
+          .version(ConfigConstants.VERSION_0_5_0)
+          .deprecated()
```

Comment:
You should use the method `alternatives`, too.

Reply:
Thx a lot.